Tasks/CopyFilesOverSSHV0/Tests/L0.ts

```diff
@@ -9,8 +10,32 @@ describe('CopyFilesOverSSHV0 Suite', function () {
     after(() => {
     });
 
-    it('Does a basic hello world test', function(done: MochaDone) {
-        // TODO - add real tests
-        done();
+    context('Utils tests: ', function () {
+        it('Should recognize UNC paths', function (done: MochaDone) {
+            const paths: string[] = [
```

Comment:
Could you please move path definitions out of L0.ts module? Maybe to some separate module for unils tests?